subsys/nrf_security/src/drivers/cracen/sxsymcrypt/src/cmdma.c

```diff
@@ -122,6 +123,7 @@ void sx_cmdma_reset(void)
 	uint32_t intrmask = sx_rdreg(REG_INT_EN);
 
 	sx_wrreg(REG_CONFIG, REG_SOFT_RESET_ENABLE);
```

Comment:
Please add a comment here to say that this is needed to avoid two short reset pulse and reference the jira task for more details.